### ISSUE TITLE
Linter doesn't warn about unnecessary usage of `^` in code

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#690]
This is my code:

```
# App.
[n] > app
  foo.plus 1 > @
  [x] > foo
    ^.n > @
```

I don't get any warnings. However, I should. The last line may be written shorter:

```
# App.
[n] > app
  foo.plus 1 > @
  [x] > foo
    n > @
```

No need to use `^` if it's not required to resolve ambiguity between names. There is only one `n` in this program, the use of `^` is not motivated.

Let's create a new lint to check this.

**Comment by yegor256** [comment]
@Snehakb1219-christ sure, please go ahead
